pom.xml

```diff
@@ -47,6 +47,16 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+             <version>2.6.1</version>
```

Comment:
No need to specify version here. Since we have spring-boot-starter-parent as the parent module. That  automatically set the compatable versions for spring boot related dependencies